llvm/test/CodeGen/AMDGPU/simple-indirect-call-2.ll

```diff
@@ -0,0 +1,66 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --check-globals
+; RUN: opt -S -mtriple=amdgcn-amd-amdhsa -passes=amdgpu-attributor %s | FileCheck --check-prefixes=CHECK,OW %s
+; RUN: opt -S -mtriple=amdgcn-amd-amdhsa -passes=amdgpu-attributor -attributor-assume-closed-world=1 %s | FileCheck --check-prefixes=CHECK,CW %s
```

Comment:
Can we make this one of those parsed, named pass parameters instead of a cl::opt

Reply:
something like `-amdgpu-attributor-closed-world`?

Reply:
No, like -passes='amdgpu-attributor<closed-world>'

Reply:
Well, it is technically feasible, but TBH I'm not sure if that's necessary. `AMDGPUAttributor` is an instance of `Attributor`, and there is already control via `cl::opt`.

Even if we do want that, a better place would be after #100953 because that introduces an extra argument which we can use `MODULE_PASS_WITH_PARAMS`.

Reply:
In general we should use fewer cl::opts

Reply:
Okay, I can stack one PR on top of #100953

Reply:
https://github.com/llvm/llvm-project/pull/101760